### ISSUE TITLE
No JSON object could be decoded

### ISSUE THREAD

**Issue by ghost** [sgricci/dewdrop#10]
Seeing the  following on startup.

```
./dewdrop/dewdrop.py:98: Warning: /tmp/buildd/glib2.0-2.36.4/./gobject/gsignal.c:2475: signal `child-added' is invalid for instance `0x176c4f0' of type `GtkMenu'
Traceback (most recent call last):
  File "./dewdrop/dewdrop.py", line 115, in show_recent
  File "./dewdrop/dapi.py", line 25, in drops
  File "./dewdrop/libdroplr.py", line 105, in list_drops
  File "./dewdrop/libdroplr.py", line 178, in execute
  File "./dewdrop/libdroplr.py", line 203, in __init__
  File "/usr/lib/python2.7/dist-packages/simplejson/__init__.py", line 451, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/dist-packages/simplejson/decoder.py", line 406, in decode
    obj, end = self.raw_decode(s)
  File "/usr/lib/python2.7/dist-packages/simplejson/decoder.py", line 426, in raw_decode
    raise JSONDecodeError("No JSON object could be decoded", s, idx)
simplejson.decoder.JSONDecodeError: No JSON object could be decoded: line 1 column 0 (char 0)
Traceback (most recent call last):
  File "./dewdrop/dewdrop.py", line 115, in show_recent
  File "./dewdrop/dapi.py", line 25, in drops
  File "./dewdrop/libdroplr.py", line 105, in list_drops
  File "./dewdrop/libdroplr.py", line 178, in execute
  File "./dewdrop/libdroplr.py", line 203, in __init__
  File "/usr/lib/python2.7/dist-packages/simplejson/__init__.py", line 451, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/dist-packages/simplejson/decoder.py", line 406, in decode
    obj, end = self.raw_decode(s)
  File "/usr/lib/python2.7/dist-packages/simplejson/decoder.py", line 426, in raw_decode
    raise JSONDecodeError("No JSON object could be decoded", s, idx)

```

**Comment by hnovaway** [comment]
I have the exact same problem. Is there any solution ?

**Comment by sgricci** [comment]
Do either (or both) of you have brand new droplr accounts?

**Comment by ghost** [comment]
I created my account the same day I installed the application. (Maybe a hour before creating this bug report.)

**Comment by sgricci** [comment]
If you go to the droplr web interface and upload something, then start the app, does it still happen?

I believe this issue is happening when it attempts to generate a menu for recent drops. 

—

**Comment by ghost** [comment]
@sgricci That does resolve the error. Thanks.

**Comment by sgricci** [comment]
Awesome!

I'll put a fix into the next release.

--Steve
On 09/27/2013 12:18 PM, Greg Fitzgerald wrote:

> @sgricci https://github.com/sgricci That does resolve the error. Thanks.
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/sgricci/dewdrop/issues/10#issuecomment-25265767.

## 

Steve Gricci
Chief Awesomenaut

"Everything was beautiful and nothing hurt."
-Kurt Vonnegut, /Slaughterhouse-Five/

**Comment by sgricci** [comment]
This has been fixed in the unstable branch and will be out with the next release. Thanks for letting me know!

**Comment by hnovaway** [comment]
Sorry for the late answer, but it solved the problem for me as well

**Comment by sgricci** [comment]
Thanks for the late answer :)

This has been fixed in the upcoming v0.10 release.
